tracing-subscriber/src/fmt/span.rs

```diff
@@ -62,8 +63,56 @@ enum State {
     Empty(usize),
 }
 
+struct SpanStackElement {
```

Comment:
Nit, take it or leave it: I don't find this name terribly descriptive; `ContextId` might have been clearer IMO. Not a big deal.

Reply:
Yes, you're right, I revert to `ContextId`